packages/monorepo-builder/packages/Release/Process/ProcessRunner.php

```diff
@@ -54,14 +54,14 @@ private function normalizeToString(string | array $content): string
     /**
      * @param string|string[] $commandLine
      */
-    private function createProcess(string | array $commandLine): Process
+    private function createProcess(string | array $commandLine, ?string $cwd = null): Process
```

Comment:
Seing the code the 2nd arg is always filled:
```suggestion
    private function createProcess(string | array $commandLine, ?string $cwd): Process
```